release/models/system/openconfig-system-bootz.yang

```diff
@@ -19,10 +19,16 @@ module openconfig-system-bootz {
     service running on a network device.";
 
```

Comment:
Can you also add a reference field to https://github.com/openconfig/bootz ?

Reply:
done.